Directory.Build.props

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <DefaultVersion>1.7.1</DefaultVersion>
+        <DefaultTargetFramework>net6.0</DefaultTargetFramework>
+        <DefaultLibraryVersion>netstandard2.0</DefaultLibraryVersion>
```

Comment:
To have .NET multi-target the build, we need to semicolon delimit these fields, after the latest push to use `TargetFrameworks` instead of `TargetFramework`.

The CI/CD to Nuget is not in the repo, so changes may need to be made there to support [multi-targeting](https://docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks)